mne/io/base.py

```diff
@@ -2331,8 +2331,6 @@ def _start_writing_raw(name, info, sel=None, data_type=FIFF.FIFFT_FLOAT,
     fid = start_file(name)
     start_block(fid, FIFF.FIFFB_MEAS)
     write_id(fid, FIFF.FIFF_BLOCK_ID)
-    if info['meas_id'] is not None:
-        write_id(fid, FIFF.FIFF_PARENT_BLOCK_ID, info['meas_id'])
```

Comment:
why this?

Reply:
@agramfort I know @bloyl is out for a bit so I'm working on this now.

Maybe we should just revert this bit for now?

Reply:
Reverted